packages/react/src/styles/_common.scss

```diff
@@ -23,7 +23,7 @@
   margin: 0; /* 2 */
 }
 
-%hiddenFromScreen {
+%visually-hidden {
   border: 0;
   clip: rect(0 0 0 0);
   clip-path: inset(50%);
```

Comment:
I wonder why this doesn't just use core's CSS? Same for buttonReset 🤔 It could be fixed in separate PR too.

Reply:
I renamed` %visuality-hidden` to `%visuallyHidden`

Reply:
I could not import core styles correctly so let's fix that in another PR.